roq-frontmatter/runtime/src/main/java/io/quarkiverse/roq/frontmatter/runtime/RoqTemplateExtension.java

```diff
@@ -46,14 +47,25 @@ public static String slugify(String text) {
     @SuppressWarnings("unchecked")
     public static List<String> asStrings(Object o) {
         if (o instanceof String i) {
-            return List.of((i).split("\\h*,\\h*|\\h{2,}"));
+            return Arrays.stream(i.split("[, \t;]"))
+                    .map(String::trim)
+                    .filter(s -> !s.isEmpty())
+                    .toList();
         }
         if (o instanceof JsonArray i) {
             return i.getList();
         }
         return List.of();
     }
 
+    public static void main(String[] args) {
```

Comment:
Could we have a test instead :)

Reply:
haha. I thought I deleted that. But yes.

Reply:
Took me a minute to figure out the import sort plugin, but I think this oughtta do it, assuming the test is in the right place. :)